src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -751,12 +775,18 @@
           }
           if (err.message === "User canceled the dialog box") {
             this.$msg.warning("Transaction en cours annulée")
-            return
+          } else {
+            this.$msg.error(
+              `Une erreur inattendue est survenue pendant le transfert d'argent. ` +
+                `Veuillez nous excuser pour la gêne occasionnée.<br>` +
+                `Vous pouvez réessayer. Si l'erreur persiste, veuillez contacter votre administrateur.`
+            )
           }
           console.log("Payment failed:", err.message)
-          throw err
+          return
```

Comment:
I'm not sure this will behave as you expected. It seems that errors won't be reset after a simple cancellation from the user.

Reply:
Ok with you ? "Resolve conversation" ?